examples/webgl_geometry_terrain_raycast.html

```diff
@@ -252,8 +252,8 @@
 
 			function onMouseMove( event ) {
 
-				mouse.x = ( event.clientX / renderer.domElement.width ) * 2 - 1;
-				mouse.y = - ( event.clientY / renderer.domElement.height ) * 2 + 1;
+				mouse.x = ( ( event.clientX * window.devicePixelRatio ) / renderer.domElement.width ) * 2 - 1;
+				mouse.y = - ( ( event.clientY * window.devicePixelRatio ) / renderer.domElement.height ) * 2 + 1;
```

Comment:
Whichever way is best. I can't do a PR ATM. Can you change it on your end?
On Oct 3, 2015 10:51 AM, "Mr.doob" <notifications@github.com> wrote:

> In examples/webgl_geometry_terrain_raycast.html
> <https://github.com/mrdoob/three.js/pull/7272#discussion_r41089725>:
>
> > @@ -252,8 +252,8 @@
> >
> >  			function onMouseMove( event ) {
> >
> > -				mouse.x = ( event.clientX / renderer.domElement.width ) * 2 - 1;
> > -				mouse.y = - ( event.clientY / renderer.domElement.height ) * 2 + 1;
> > +				mouse.x = ( ( event.clientX * window.devicePixelRatio ) / renderer.domElement.width ) * 2 - 1;
> > +				mouse.y = - ( ( event.clientY * window.devicePixelRatio ) / renderer.domElement.height ) * 2 + 1;
>
> @spite <https://github.com/spite> ping?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrdoob/three.js/pull/7272/files#r41089725>.
>